.gitignore

```diff
@@ -0,0 +1,5 @@
+*.DS_Store
```

Comment:
Good to add package-lock.json here. It doesn't really belong in an npm module, only a project.